timelock-server/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosTimeLockServer.java

```diff
@@ -270,10 +301,7 @@ private ManagedTimestampService createPaxosBackedTimestampService(String client)
 
         PaxosSynchronizer.synchronizeLearner(ourLearner, learners);
 
-        return AwaitingLeadershipProxy.newProxyInstance(
-                ManagedTimestampService.class,
-                () -> createManagedPaxosTimestampService(proposer, client, acceptors, learners),
-                leaderElectionService);
+        return () -> createManagedPaxosTimestampService(proposer, client, acceptors, learners);
     }
```

Comment:
yeah, note that i renamed this method to `createRawPaxosBackedTimestampServiceSupplier`